### PR TITLE
customers#45 Add splitChunks in prod mode

### PR DESCRIPTION
**Pull request by oubchid**
related to https://github.com/linagora/customers/issues/45

### CODE REVIEW

**Review by asmaNode — APPROVED** [review]

File: webpack.prod.js [hunk 1]
```diff
@@ -42,12 +42,25 @@ module.exports = merge(commons, {
...
+      chunks: 'all',
+      maxInitialRequests: Infinity,
+      minSize: 200000,
+      cacheGroups: {
+        vendor: {
+          test: /[\\/]node_modules[\\/]/,
+          name(module) {
+            const packageName = module.context.match(/[\\/]node_modules[\\/](.*?)([\\/]|$)/)[1];
+
+            return `${packageName.replace('@', '')}`;
```
Comment (alagane) [inline webpack.prod.js:56]: Why is this needed?

Comment (oubchid) [reply webpack.prod.js:56]: Same server can't serve files with an @ in the name

**Review by alagane — APPROVED** [review]
